go.mod

```diff
@@ -34,3 +34,5 @@ require (
 	gopkg.in/ini.v1 v1.55.0 // indirect
 	gopkg.in/yaml.v2 v2.2.4
 )
+
+replace github.com/minio/minio => ../minio
```

Comment:
We don't need this line, right?

Reply:
You need the minio changes to go along with this. That is in PR review, so this is needed until then